From 9ca39c9cd1391f59ebaf4577239a7c657d9300c0 Mon Sep 17 00:00:00 2001
From: Ankur Verma <ankur.verma@outlook.com>
Date: Wed, 9 Jun 2021 01:07:37 -0700
Subject: [PATCH] Fix android

---
 src/rtl_adsb.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/rtl_adsb.c b/src/rtl_adsb.c
index 7aea8dd..dd112b9 100644
--- a/src/rtl_adsb.c
+++ b/src/rtl_adsb.c
@@ -492,7 +492,6 @@ int main(int argc, char **argv)
 	else {
 		fprintf(stderr, "\nLibrary error %d, exiting...\n", r);}
 	rtlsdr_cancel_async(dev);
-	pthread_cancel(demod_thread);
 	pthread_join(demod_thread, NULL);
 	pthread_cond_destroy(&ready);
 	pthread_mutex_destroy(&ready_m);
-- 
2.31.1

